src/rgw/driver/rados/rgw_bucket_sync.cc

```diff
@@ -494,14 +524,15 @@ bool RGWBucketSyncFlowManager::allowed_data_flow(const rgw_zone_id& source_zone,
                                                  bool check_activated) const
 {
   bool found = false;
-  bool found_activated = false;
+  bool found_activated = true;
```

Comment:
this would return `true` if `flow_groups` are empty. However, I can't find anywhere were it's even called with `check_activated` true, so maybe a moot point. Am I missing something?